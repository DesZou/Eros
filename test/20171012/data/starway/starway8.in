901422 746836 1000
650952 655274
465893 368265
306502 143181
176233 408899
365720 60339
227730 25119
726860 181464
44070 537260
749450 492763
31015 154774
458487 269615
445703 435975
863453 87793
161446 337689
731032 550220
520941 286876
519923 435082
634709 201353
645084 673327
386429 727968
477398 353367
180436 346201
881006 652239
303260 210036
513824 400298
556520 224608
217889 354364
294838 465338
144656 90927
554485 137200
381188 688873
566396 14123
80912 136591
686637 515148
843601 171505
504548 304736
658402 584910
482252 568503
184345 657081
799467 571161
566109 658392
754269 473391
761747 345360
813832 141630
375632 72777
573661 693679
386264 385423
775227 565750
87132 165400
86689 185397
246801 726850
739537 77160
404026 629843
137251 339905
240852 559876
16014 30471
157994 163796
310225 283469
527116 281267
249776 348563
774983 469286
177713 365475
56566 451838
408824 4662
203569 461539
646212 726294
639625 182955
72713 528163
435187 112430
485988 28453
400802 316008
475035 1767
409556 608836
1075 252504
534729 344182
446469 15977
456861 342967
174730 582551
608310 361581
54706 246523
851391 168463
448323 671703
825764 364241
131474 132983
568129 505836
505909 112612
264426 390901
793369 19360
281339 118718
17616 558048
106842 370689
798006 92230
514951 200785
546959 437581
594373 149353
331960 3948
683616 66051
776216 380594
836692 544856
465182 655819
686973 372725
539039 66022
89828 598272
333902 584425
759926 492217
406011 214597
340283 576640
655465 115277
847450 297248
752219 159755
82919 740721
337134 411293
401991 477111
716129 637975
265981 105927
799316 505901
6859 502287
21964 724689
849252 30557
757886 654345
667217 630390
486720 67231
637933 350572
127635 480637
328843 546221
301817 196432
842755 29252
822113 75348
497480 591933
812887 613713
451236 380785
531712 383513
803508 222576
886613 72660
84581 169957
784655 554605
19272 650498
351186 269859
438209 697777
589379 240567
30340 226196
43452 486436
251231 51233
48514 178483
245430 235030
508888 29066
24700 313211
814502 312624
703056 260189
12891 632737
737796 239317
339832 523468
170262 292486
399490 519156
686645 93650
251487 732705
345643 585761
471235 290801
666871 75770
808020 311813
568855 148641
607575 456013
46158 62145
763948 83968
689022 230150
663359 164398
543047 546306
446725 343921
464628 414299
618081 631159
36535 8404
457721 318752
299990 32663
348571 420184
820169 418773
681169 283054
666904 348828
861971 542200
7848 232690
260578 197144
809291 103998
742323 508934
750849 114372
33634 392015
727242 401478
475804 51596
282219 587441
123604 312840
345975 253719
241153 57499
531110 408605
233768 377535
234561 571915
538955 387587
285682 386805
488140 679297
579008 293642
476005 558411
655040 393274
242593 553664
79705 238049
711425 189976
649641 471456
147063 105415
860657 523721
138730 475404
567725 605435
260189 427278
381914 656999
705105 547319
159998 388304
537812 563278
359961 149947
61973 14696
896450 564771
496763 367792
502380 294604
252653 540722
731392 277776
679248 493880
838278 455043
176527 560094
116408 287347
470587 76137
132412 481163
732462 377845
632853 729617
17866 574487
121069 131148
524549 202895
399684 389047
637012 722602
194110 463866
434162 101139
465631 90158
456115 256297
2645 165621
799821 460294
276411 392867
828251 599915
663650 677320
32511 27705
764555 581555
551649 382370
870875 94865
12233 348699
164627 109134
840870 133168
754503 733478
481430 716349
842509 100765
207102 581384
254741 344006
556052 589931
523920 73497
200741 33117
637676 553011
90314 521586
603842 336779
423433 481435
458261 584850
295586 70232
129553 339857
674653 343241
308466 624255
879179 392686
320207 88843
504013 327014
676533 32023
441957 63762
612526 658548
257845 373821
410014 259528
336616 321850
457267 490750
101934 661654
130281 49930
371378 707764
159953 716656
559747 552928
35153 557441
899870 235014
591283 618576
229977 604916
371013 19744
291634 566320
600014 270051
358297 12107
205055 476907
278826 59285
280367 150312
261090 661795
785605 269219
520493 646535
320581 283934
519177 659853
647556 599928
63120 701068
497077 416262
462363 206525
356648 4767
482211 694198
189961 145507
76271 312584
785029 302269
550183 118733
185452 593135
416717 408998
422916 629460
493633 129767
170575 151889
288372 226696
91237 196667
46576 283147
547356 397030
767258 310058
794856 258355
271989 102931
321906 216812
645307 730802
896054 672787
822143 18634
174145 387119
880724 328892
691425 444288
376316 233710
363382 227633
163926 240484
852959 244990
44803 299515
4172 303449
688344 589897
155813 304905
291069 99799
689382 344280
94117 554921
519232 355009
738161 1857
695215 247676
196321 575090
474515 426854
790830 107808
290378 488019
446981 389126
891685 568532
541356 312171
737292 683485
544561 743913
515762 116403
778229 41339
711143 232681
19630 471763
563688 584049
579784 101165
328699 297512
451924 687692
596063 675776
86104 96759
632545 692311
732386 734891
878607 488986
142089 579178
364581 395672
163956 111704
25618 397607
630140 373426
737057 727965
215175 91539
741938 146182
768502 16634
168890 546742
696288 742328
227753 102258
556982 466838
686804 155526
337589 201875
328358 5030
745407 568517
400604 580387
365431 421713
89130 606614
684236 296343
225040 107554
617238 157367
768122 261446
784776 459278
60659 152289
198430 197565
697856 420773
651728 493707
49080 67535
258819 735398
765630 567706
673235 603531
541838 662749
890179 623482
356884 463685
754364 230023
624187 361986
41411 209941
465459 402124
618933 280817
738329 440184
552536 126894
33349 545602
683016 32197
428998 404922
725679 532744
131143 132010
696531 267476
713321 590578
183062 615018
284882 707452
611308 192284
681119 115636
153019 41216
770914 613663
138164 637813
370728 386010
851656 89784
278318 415062
388377 460812
601327 453962
137851 617449
369676 100088
748278 584279
901198 743334
362073 365675
64062 259348
564865 80584
772093 729646
477096 313569
274558 232579
673092 3545
357499 132876
526374 279817
637838 671057
715284 269745
429702 258962
870150 141453
870956 65443
532749 628277
456257 140448
164906 387893
711752 619626
126352 377405
207599 697600
443176 133447
864208 706967
572794 121432
348849 457100
557151 27091
437402 4423
93837 261000
744081 594492
441705 515130
838277 124571
122983 357209
101408 337634
557030 357307
451899 377055
552329 730802
86565 579242
384686 723386
901221 185815
79715 385054
768700 578158
413221 240867
556478 258973
800155 208321
772432 216777
670510 123301
810836 372705
866961 272947
109647 305075
704477 9940
722144 523865
363475 59133
483677 353682
58396 348953
509997 619561
860314 121519
580033 50643
846500 80786
441666 449991
383778 185745
739892 574960
746796 311908
167841 356350
414698 517759
489496 514113
645371 658803
85714 339557
512653 685742
164951 364054
104906 137599
363556 245272
740581 688505
488385 190616
128234 742348
255455 524471
134612 9836
697859 216883
544254 213252
166198 546391
477245 649256
103958 238139
850882 376345
368180 330711
276652 517577
520283 414422
626235 227454
72756 584929
117386 594335
59223 30924
607020 349907
515761 341012
501797 40883
722904 256644
518451 703663
1782 344376
233338 719612
701587 514991
192139 111792
462037 286579
461115 62666
600178 679631
30814 232913
650310 672543
702157 298597
844384 271325
29877 285683
213183 669641
882286 241506
603756 315815
759761 681880
601836 235690
77036 188014
269086 105484
843905 425650
835192 417574
413873 209821
610253 443570
6060 143129
671069 342821
525860 15112
334906 218583
829250 622907
541742 55581
548923 724569
406333 422770
110445 130019
642189 742110
20677 77118
474641 666628
465018 344687
174614 695248
463961 428223
251698 733837
233057 686207
447212 192238
382633 67019
223585 102831
522619 567010
732923 138178
820201 239631
749201 509647
584086 313486
732427 159440
772437 274682
246305 485219
209090 163281
269011 584771
324789 398115
663083 560707
410814 505289
586471 130576
832905 440928
826947 648232
761013 413590
532687 56293
886615 203243
852266 141775
692355 315558
686426 436707
779327 575158
778538 442246
790135 324780
753659 195318
352461 656694
889792 712139
345487 99998
722316 311746
642882 354439
899312 254803
288702 213644
707872 606889
62066 246527
853156 369897
483365 424777
774626 727185
398607 20607
57296 530728
798499 105660
545356 86773
805336 259559
351080 168265
61399 277297
815827 401176
335802 394566
20987 455113
339751 733728
731699 668092
512169 73288
447290 384969
771410 275329
465633 212228
248665 337549
470316 689964
198954 725300
264808 373483
57192 637054
456461 487305
693298 457893
718899 440797
82642 526243
785428 717138
249531 268737
60841 177039
634853 78097
888465 106543
512017 168818
640011 650142
191776 669730
343795 13414
522726 106081
97591 258444
506748 299076
370623 682643
685295 546296
105083 187013
609197 733846
308379 707094
488706 375679
582809 278941
190885 556217
17833 658722
331911 162334
171862 193311
55848 619172
184138 38748
763265 27011
684902 426064
137623 585145
526118 170399
856662 592478
100803 339026
351212 135628
466506 551839
170295 488664
164585 226576
740382 735145
53648 565799
682213 128413
754444 479063
563089 265765
888501 304607
550086 163836
880911 63386
673 201835
374596 655064
141361 571561
16769 127750
899949 82364
101642 157345
732427 246979
235708 694348
821842 510751
137640 711924
80676 458641
427993 516177
116210 479771
747607 378922
234789 744738
696449 551185
853066 291636
807120 197404
627362 230803
600973 509333
44435 101674
544131 453856
636135 714060
199813 147077
102510 705607
172064 415247
229647 345861
404842 392467
732056 215581
684613 562310
317592 637433
57120 630006
378645 587804
807063 72944
313793 381813
10037 194152
775087 603969
237143 730745
338201 529211
192301 321695
1461 303628
166974 559592
151766 215763
288690 529618
620691 717704
621891 310389
560582 409740
742899 42143
827228 55493
692085 324365
681786 706482
81079 673236
272014 479080
767290 452188
670884 150097
263006 41155
233091 212618
430999 233449
436510 439408
883020 283221
483776 337757
237792 85898
218844 3101
503655 461174
810313 606114
782549 481014
93116 379088
390552 78271
457864 553809
91800 746386
129832 577952
556960 603147
808175 676178
771935 444088
175799 283157
861869 718614
795472 594527
753810 615050
899353 478355
632088 118233
7768 436363
37369 3399
269962 353333
645361 373231
802310 577388
101483 31302
23171 205895
274517 314254
129846 529315
707357 113098
111332 691285
474003 460904
37550 370886
850897 177957
872214 343000
715000 451263
371531 452081
356572 514568
440293 434273
118410 699823
312682 561514
317209 597082
875896 450523
663000 296394
517051 575206
660369 348133
856508 674953
246528 5357
672993 104338
294378 207909
661766 700169
654198 644016
533351 571712
30479 299436
181663 324469
891004 353278
228503 20124
726469 444480
480472 134419
861349 617633
731375 108128
79839 741170
622363 258296
666323 82894
674343 29827
466312 744413
88189 687754
815605 17079
229854 827
567167 646056
11572 237944
524099 297129
562208 219707
559175 682312
766564 687786
421040 367144
850158 461007
116432 170850
757088 513043
265633 361511
214040 524700
519442 236000
161057 73237
296824 48386
313806 362580
47651 104638
665023 717464
775440 151839
74933 394060
77520 109628
497449 424952
900652 410962
559101 499290
185559 598198
681451 301935
319876 390081
100333 250429
824385 652488
896752 173609
396340 379349
376657 404226
503079 42059
530330 129746
51730 175469
762891 338219
139355 265322
169570 578654
582353 614757
826695 634938
285501 390758
112359 238177
882686 606807
826129 371495
135554 349693
880884 379195
772136 13601
375514 153572
772193 656413
405047 320316
380152 138109
237538 83230
405733 149840
717686 61973
6167 378321
580358 208666
418282 194384
186945 674558
163321 163037
204579 438469
480758 24140
291292 92800
248543 236283
354950 202971
645669 377773
433346 601513
516597 522554
438178 60235
838262 589065
303522 684649
839487 10061
234149 358050
549011 548650
558056 6347
442762 42471
549612 414987
96578 653737
215775 297584
496674 209500
191735 7555
700295 583135
443308 644601
166837 429473
441104 320854
760908 217603
42998 83125
213152 303800
720983 253867
874133 180338
644711 635850
682246 30108
88482 208829
54772 37635
210322 217724
438211 324240
428090 454631
8039 174736
605938 230856
744319 603693
804680 483517
296406 518500
636765 327970
467547 329478
20991 285663
487153 317779
689423 100437
814277 563524
816639 42772
617137 124110
817698 330169
634647 615037
592716 743702
558089 668577
95894 687428
277499 237268
709395 117478
607829 173075
744030 327370
560506 78772
317957 79853
748643 506989
591706 617776
33892 12407
687625 539313
279089 284980
670267 566495
670100 145235
861898 398950
28360 23124
58978 71642
701929 591673
88544 73560
261067 60009
529714 93829
556913 493185
726979 385313
757431 254859
490026 691886
52942 253537
176512 635910
808278 517469
408398 113919
233814 115620
686857 608963
776581 215339
32466 594929
808048 156141
741207 188134
12258 206301
588346 315122
345675 393179
334738 126645
805650 383939
385844 348177
187335 74294
762380 455279
465104 309496
666331 416383
171600 641683
474194 508727
900204 544519
769437 455364
81536 516426
353793 209902
610386 243176
809712 636477
784424 620962
281972 618167
146747 177054
26891 356178
78615 718783
746204 150098
721170 642236
518398 288558
546701 302224
270121 62923
403086 223874
504375 609343
174768 209402
822783 400390
756786 415991
73485 461692
819277 595926
118324 66743
178751 44678
72383 456744
334275 310087
58991 3780
730585 298462
593695 527499
282270 544757
483289 652688
793424 310869
668227 209555
353274 616738
392306 388352
703587 209870
359051 378270
391896 306010
396448 671374
737941 626542
578121 25372
