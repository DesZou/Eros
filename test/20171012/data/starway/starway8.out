14197.795832452
